src/app/components/nav/footer/footer.component.scss

```diff
@@ -11,7 +11,7 @@
   overflow-x: hidden;
   background-repeat: no-repeat;
   background-size: 100% 100%;
-  z-index: 10;
+  z-index: 0;
```

Comment:
Why are you decreasing footer's `z-index`, increase for the button.

Reply:
Because scroll button is child of  "<div class="privacy-content>" and <app-footer> is sibling of that div.

Reply:
`<app-footer>`